src/System.Net.HttpListener/src/System/Net/Windows/ContextFlags.cs

```diff
@@ -66,5 +66,5 @@ internal enum ContextFlags
         AllowMissingBindings = 0x10000000,   // ASC_REQ_ALLOW_MISSING_BINDINGS
 
         UnverifiedTargetName = 0x20000000,   // ISC_REQ_UNVERIFIED_TARGET_NAME
-    }
+    }*/
```

Comment:
If it's actually dead code, the whole file should be deleted rather than leaving the code in but commented out.